test/verify/check-storage-format

```diff
@@ -92,6 +92,39 @@ class TestStorageFormat(StorageCase):
         else:
             check_type("ntfs", 128)
 
+    def testFormatCandel(self):
```

Comment:
typo - s/candel/cancel

Reply:
You found it! :-) This typo was just too cute and seasonal to fix immediately, imo... :-)

Reply:
Fixed.